app/modules/import-export/processes/import.php

```diff
@@ -698,7 +700,9 @@ private function add_import_session_option( $start_time ) {
 
 		$import_sessions[ $this->session_id ] = [
 			'session_id' => $this->session_id,
-			'kit_name' => $this->manifest['name'],
+			'kit_title' => $this->manifest['title'] ?? '',
+			'kit_name' => $this->manifest['name'] ?? '',
+			'kit_thumbnail' => $this->manifest['thumbnail'] ?? '',
 			'kit_source' => $this->settings_referrer,
 			'user_id' => get_current_user_id(),
```

Comment:
Can you combine this array with the one at line 386? maybe a common function?

Reply:
I changed the logic a little bit and merged them.